src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+    private List<Integer> numbers;
+
+    public Lotto(LottoNumbers numbers) {
+        this.numbers = numbers.generateRandomNumbers();
+    }
+
+    public int getMatchCount(LottoNumbers numbers) {
+        return (int) numbers.getWinningNumbers().stream().filter(n -> this.numbers.contains(n)).count();
+    }
```

Comment:
```suggestion
    public int getMatchCount(Lotto lotto) {
        return (int) lotto.numbers.stream()
                        .filter(numbers::contains)
                        .count();
    }
```
- LottoNumbers를 받는게 아니라, Lotto를 받으면 어떨까요?
- 이럴 경우 위와 같이 표현할 수 있습니다.
- 그리고 가독성을 위해서 Stream method는 한 줄애 한 개만 사용해주세요.

Reply:
Stream 사용이 익숙하지 않았는데 해당 코멘트 참고해서 많은 도움됐습니다👍